app/jobs/people/cleanup_job.rb

```diff
@@ -35,10 +33,6 @@ def event_participation_in_cutoff?(person)
          .any?
   end
 
-  def ignore_person?(person)
```

Comment:
Hast du kontrolliert dass das in keinem Wagon überschrieben wird?

Reply:
Es wird nicht überschrieben.